Sources/Brave/Frontend/Browser/TabManager.swift

```diff
@@ -416,12 +416,30 @@ class TabManager: NSObject {
     }
     // When bulk adding tabs don't notify delegates until we are done
     self.isRestoring = true
-    var tab: Tab!
-    for url in urls {
-      tab = self.addTab(URLRequest(url: url), flushToDisk: false, zombie: zombie, isPrivate: isPrivate)
+    let tabs = urls.map {
+      let tab = self.addTab(URLRequest(url: $0),
```

Comment:
Its a bit weird to have side-effects (addTab which alters the state of the class) trigger inside of a `map` which is just supposed to transform data from 1 type to another